### DISCUSSION TITLE
HAM + Lambs AI?

### DISCUSSION THREAD

**Discussion by OverlordZorn** [Vdauphin/HeartsAndMinds#1521] [Question and answer] [answered]
How well does HeartsAndMinds and Lambs AI Mods play together?

**Comment by Vdauphin** [comment, marked as answer]
Hello,

If not already, you may want to look here
https://github.com/Vdauphin/HeartsAndMinds/discussions/1077
And here:
https://github.com/Vdauphin/HeartsAndMinds/pull/1505

Cheers
